boot.php

```diff
@@ -15,9 +20,9 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+$filename = getcwd().'/'.$argv[1];
```

Comment:
```getcwd()``` isn't a good idea because if the user specifies a full path it will not work and cause confusion to a user. Yes, loading a ROM by name alone makes it easy, but it doesn't mean it's the correct way to handle input.